addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -401,6 +401,10 @@
 
     .o_data_row.o_selected_row > .o_data_cell.o_invalid_cell:not(.o_readonly_modifier) {
         border-bottom: 1px solid $danger;
+
+        &:not(:focus-within) {
+            background-color: rgba($o-danger, .1);
```

Comment:
We should use the --o-input-background-color CSS variable instead

Reply:
We are targeting the invalid_cell otherwise the danger background will be applied only on the input (which doesn't take the full height of the cell)